analyses/snv-callers/README.md

```diff
@@ -12,14 +12,15 @@ The GDC has [good documentation on the fields](https://docs.gdc.cancer.gov/Data/
   * [Genomic regional analyses](#genomic-regional-analyses)  
   * [Tumor mutation burden](#tumor-mutation-burden-calculation)
   * [COSMIC mutation overlap](#cosmic-mutation-overlap)  
-* Comparison analysis - in development
+* [Comparison analysis](#comparison-of-callers)
  * [Mutation IDs](#mutation-ids)
+ * [Combinations of callers]()
```

Comment:
No link?

Reply:
This section shouldn't be here until a later PR, so I will remove it from the TOC.